programs/conditional_vault/src/error.rs

```diff
@@ -2,14 +2,34 @@ use super::*;
 
 #[error_code]
 pub enum VaultError {
+    #[msg("An assertion failed")]
+    AssertFailed,
     #[msg("Insufficient underlying token balance to mint this amount of conditional tokens")]
     InsufficientUnderlyingTokens,
+    #[msg("Insufficient conditional token balance to merge this `amount`")]
+    InsufficientConditionalTokens,
     #[msg("This `vault_underlying_token_account` is not this vault's `underlying_token_account`")]
     InvalidVaultUnderlyingTokenAccount,
     #[msg("This conditional token mint is not this vault's conditional token mint")]
     InvalidConditionalTokenMint,
-    #[msg("Vault needs to be settled as finalized before users can redeem conditional tokens for underlying tokens")]
+    #[msg("Question needs to be resolved before users can redeem conditional tokens for underlying tokens")]
```

Comment:
It's nitpicking but the idea passing or failing a question is funny / odd

Reply:
In this case, the question is "Will proposal x pass or fail?" and you're resolving it to "Pass" or "Fail"